src/sentry/static/sentry/app/components/group/suggestedOwners.jsx

```diff
@@ -20,6 +20,7 @@ const SuggestedOwners = React.createClass({
       selector: '.tip',
       html: true,
       container: 'body',
+      template: '<div class="tooltip" role="tooltip"><div class="tooltip-arrow"></div><div class="tooltip-inner owners"></div></div>',
```

Comment:
re: `class` Does this not need to be jsx? 

Also, I've been bitten by overflowing content several times lately. Maybe we should eventually handle this at a higher level, but in the meantime can you add a `.break-word` to `.tooltip-inner` ?